frontend/src/components/CaseContainer.js

```diff
@@ -124,7 +124,8 @@ class CaseContainer extends Component {
 
     let arrow = " --- ";
     /// Recursive function to go down the tree adding components
-    function addTree(itemType, parent, parentNode, outputmd) {
+    function addTree(itemType, parent, parentNode, outputmd, visited) {
```

Comment:
what does the "visited" do?   Is this because when we have multiple parents, we only want one of them to add a new node?

Reply:
Yes, that's exactly it. Without that, any items downstream from multiple parents got the same lines drawn multiple times.